components.json

```diff
@@ -457,6 +457,23 @@
 			"title": "Git",
 			"owner": "lgiraudel"
 		},
+		"glimmer": {
+			"title": "Glimmer / Ember",
+			"require": "markup-templating",
+			"owner": "NullVoxPopuli",
+			"alias": ["hbs", "handlebars", "htmlbars"],
+			"aliasTitles": {
+				"hbs": "Handlebars",
+				"handlebars": "Handlebars",
+				"htmlbars": "HTMLBars"
```

Comment:
Why? We already have a language for Handlebars.

Reply:
most ember/glimmer developers have been using `hbs` as a backstop for lack of better syntax support. By including the glimmer syntax instead of the handlebars syntax in their bundles, they'll get new highlights without having to change any of their code

Reply:
Aliases are supposed to be either 1) alternative names referring to the same language (e.g. `js` and `javascript`) or 2) other languages that use the same (or only a subset of the) syntax (e.g. `svg` and `markup`).

I've seen that the Ember docs use `handlebar` but this is something they have to fix. We only provide languages (and syntax highlighting, I guess). Please remove the aliases.

However, these aliases aren't bad just misplaced. Do you want to make a separate PR adding these aliases to Handlebar? Otherwise, I'll do that.